src/FESpaces/FESpaceFactories.jl

```diff
@@ -1,8 +1,8 @@
 
 function FESpace(
   model::DiscreteModel,
-  cell_fe::CellFE;
-  conformity=nothing,
+  cell_fe::CellFE,
+  cell_conformity::CellConformity=CellConformity(cell_fe);
```

Comment:
I would not include a positional argument here. `CellConformity` is in `CellFE` and I don't think we want/consider the case in which we have different `CellConformity` in `CellFE` than the one passed in this constructor.

Reply:
ok, agreed. I do not see either why this is necessary, and can be even contraproductive.